suite/testcommon.py

```diff
@@ -883,6 +883,56 @@ def get_functions(self, bv):
     def get_comments(self, bv):
         return bv.functions[0].comments
 
+    def test_possiblevalueset_parse(self):
+        """ Failed to parse PossibleValueSet from string"""
+        file_name = self.unpackage_file("helloworld")
+        try:
+            bv = binja.BinaryViewType.get_view_of_file(file_name)
```

Comment:
use 
```
with binja.BinaryViewType.get_view_of_file(file_name) as bv:
```
or better yet
```
with binja.open_view(file_name) as bv:
```
as these implicitly execute `bv.file.close()` when the context ends.